spec/models/users/setting_spec.rb

```diff
@@ -4,34 +4,14 @@
   describe "validations" do
     subject { setting }
 
-    let(:setting) { create(:users_setting, user: user) }
     let(:user) { create(:user) }
+    let(:setting) { user.setting }
 
     it { is_expected.to validate_length_of(:inbox_guidelines).is_at_most(250).allow_nil }
     it { is_expected.to define_enum_for(:inbox_type).with_values(private: 0, open: 1).with_suffix(:inbox) }
-    it { is_expected.to define_enum_for(:config_font).with_values(default: 0, comic_sans: 1, monospace: 2, open_dyslexic: 3, sans_serif: 4, serif: 5) }
-    it { is_expected.to define_enum_for(:config_navbar).with_values(default_navbar: 0, static_navbar: 1) }
-    it { is_expected.to define_enum_for(:config_theme).with_values(default_theme: 0, minimal_light_theme: 1, night_theme: 2, pink_theme: 3, ten_x_hacker_theme: 4) }
-
-    describe "when validating feed_url", vcr: true do
```

Comment:
This is the associated spec for the `validate_feed_url` private method that I deleted in `users/setting.rb`